docs/run-locally.md

```diff
@@ -96,4 +96,25 @@ environment.
 Now it is possible to run HCO from VS Code.
 
 ![](../images/run_local_from_vscode.png)
- 
\ No newline at end of file
+ 
```

Comment:
General commment about this readme - since we remove the HC CR from the deployment, please add instruction how to deploy it.

Reply:
done